orbit/hyperspace.py

```diff
@@ -47,18 +47,22 @@ def do_query_username(args):
 def do_validate_token(args):
     need(args, t=True)
 
-    ses = db.ses_getby_token(args.token)[0]
+    ses = db.Session.get_or_none(db.Session.token == args.token)
     if ses:
-        print(ses[1])
+        print(ses.username)
     else:
         print('null')
 
 
 def do_drop_session(args):
     need(args, u=True)
-    dropped = db.ses_delby_username(args.username)[0]
-    if dropped:
-        print(dropped[0])
+    query = (db.Session
```

Comment:
might be cleaner to put this in a helper function since it could be used elsewhere

Reply:
see my response to other one.